src/repository/storage/postgres/orm/sequelize/note.ts

```diff
@@ -195,20 +218,42 @@ export default class NoteSequelizeStorage {
   /**
    * Gets note list by creator id
    *
-   * @param creatorId - note creator id
+   * @param userId - id of certain user
    * @param offset - number of skipped notes
    * @param limit - number of notes to get
-   * @returns { Promise<NoteList> } note
+   * @returns list of the notes
    */
-  public async getNoteListByCreatorId(creatorId: number, offset: number, limit: number): Promise<Note[]> {
-    return await this.model.findAll({
+  public async getNoteListByUserId(userId: number, offset: number, limit: number): Promise<Note[]> {
+    if (this.visitsModel === null) {
+      throw new DomainError('NoteVisit model should be defined');
+    }
+
+    const reply = await this.model.findAll({
       offset: offset,
       limit: limit,
       where: {
-        creatorId,
+        '$noteVisits.user_id$': userId,
```

Comment:
why not just userId?

Reply:
this is sequelize syntax for accessing an included model